lib/src/package_context.dart

```diff
@@ -132,6 +136,13 @@ class PackageContext {
     }
   }
 
+  Future<List<ScreenshotResult>> processScreenshots(
+      List<p.Screenshot> screenshots, String pkgDir) async {
+    if (_screenshotResults != null) return _screenshotResults!;
+    _screenshotResults = await processAllScreenshots(screenshots, pkgDir);
+    return _screenshotResults!;
```

Comment:
nit: Can't we avoid two return statements with;
```suggestion
    if (_screenshotResults == null) {
       _screenshotResults = await processAllScreenshots(screenshots, pkgDir);
    }
    return _screenshotResults;
```

or maybe just:


```suggestion
    return _screenshotResults ??= await processAllScreenshots(screenshots, pkgDir);
```

Reply:
Updated.